packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -152,8 +152,12 @@ export abstract class SharedObjectCore<
 		this.mc = loggerToMonitoringContext(this.logger);
 
 		const { opProcessingHelper, callbacksHelper } = this.setUpSampledTelemetryHelpers();
-		this.opProcessingHelper = opProcessingHelper;
-		this.callbacksHelper = callbacksHelper;
+		// Non null asserting here since opProcessingHelper is defined in setUpSampledTelemetryHelpers
+		// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+		this.opProcessingHelper = opProcessingHelper!;
+		// Non null asserting here since callbacksHelper is defined in setUpSampledTelemetryHelpers
+		// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+		this.callbacksHelper = callbacksHelper!;
```

Comment:
If this is always true, then prefer to assert inside setUpSampledTelemetryHelpers instead, so that it has a stronger return type.